mmv1/products/firebasedatabase/api.yaml

```diff
@@ -0,0 +1,91 @@
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Product
+name: FirebaseDatabase
+display_name: Firebase Realtime Database
+versions:
+  - !ruby/object:Api::Product::Version
+    name: beta
+    base_url: https://firebasedatabase.googleapis.com/v1beta/
+scopes:
+  - https://www.googleapis.com/auth/cloud-platform
+apis_required:
+  - !ruby/object:Api::Product::ApiReference
+    name: Firebase Realtime Database API
+    url: https://console.cloud.google.com/apis/library/firebasedatabase.googleapis.com/
+objects:
+  - !ruby/object:Api::Resource
+    name: 'Instance'
+    min_version: beta
+    base_url: projects/{{project}}/locations/{{region}}/instances/
+    self_link: projects/{{project}}/locations/{{region}}/instances/{{instance_id}}
+    create_url: projects/{{project}}/locations/{{region}}/instances?databaseId={{instance_id}}
+    description: A Firebase Realtime Database instance.
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      guides:
+        'Official Documentation':
+          'https://firebase.google.com/products/realtime-database'
+      api: 'https://firebase.google.com/docs/reference/rest/database/database-management/rest'
+    parameters:
+      - !ruby/object:Api::Type::String
+        name: 'region'
+        description: |
+          A reference to the region where the Firebase Realtime database resides.
+        required: true
+        input: true
+        url_param_only: true
+      - !ruby/object:Api::Type::String
+        name: 'instance_id'
+        description: |
+          The globally unique identifier of the Firebase Realtime Database instance. 
+          Instance IDs cannot be reused after deletion.
+        required: true
+        input: true
+        url_param_only: true
+    properties:
+      - !ruby/object:Api::Type::String
+        name: name
+        description: |
+          The fully-qualified resource name of the Firebase Realtime Database, in the
+          format: projects/PROJECT_NUMBER/locations/REGION_IDENTIFIER/instances/INSTANCE_ID
+          PROJECT_NUMBER: The Firebase project's [`ProjectNumber`](https://firebase.google.com/docs/projects/api/reference/rest/v1beta1/projects#FirebaseProject.FIELDS.project_number)
+          Learn more about using project identifiers in Google's [AIP 2510 standard](https://google.aip.dev/cloud/2510).
+        output: true
+      - !ruby/object:Api::Type::String
+        name: database_url
+        description: |
+          The database URL in the form of https://{instance-id}.firebasio.com for us-central1 instances
```

Comment:
Just wanted to check, is `firebasio.com` correct? I thought it might be a typo intended to be `firebaseio.com`.